src/main/java/com/osiris/dyml/YamlSection.java

```diff
@@ -166,41 +166,52 @@ public YamlSection addKeys(String... keys) {
         return this;
     }
 
+    /**
+     * See {@link #addValues(List)} for details.
+     */
+    public <T> YamlSection addValues(T... v) {
+        if (v != null)
+            return addValues(utils.arrayToValuesList(v));
+        else
+            return addValues((List<SmartString>) null);
+    }
+
     /**
      * See {@link #addValues(List)} for details.
      */
     public YamlSection addValues(String... v) {
```

Comment:
this function can be removed since we got the generic function above,
as well as the other related functions that take Strings